apps/web/components/organisms/pagination/pagination.tsx

```diff
@@ -0,0 +1,92 @@
+import { useRouter } from 'next/router';
+import { PaginationItem } from 'molecules/paginationItem/paginationItem';
+import { PaginationArrow } from 'molecules/pagginationArrow/paginationArrow';
+
+type PaginationProps = {
+	numberOfItems: number;
+};
+
+export const Pagination = ({ numberOfItems }: PaginationProps) => {
+	const router = useRouter();
+	const currentPage: number =
+		router.query.page && !isNaN(Number(router.query.page))
+			? Number(router.query.page)
+			: 1;
+	const perPage =
+		router.query.page && !isNaN(Number(router.query.limit))
+			? Number(router.query.limit)
+			: 11;
```

Comment:
What is this number "11" ?

Reply:
Could be moved to constant `SOMETHING_NUMBER`

Reply:
> What is this number "11" ?

It's default quantity of items per page